include/onnc/Analysis/GlobalStatistics.h

```diff
@@ -0,0 +1,30 @@
+//===- SkyGlobal.h --------------------------------------------------------===//
+//
+//                               Skymizer
+//
+// Copyright (C) 2013, Skymizer Inc.. All rights reserved.
+//
+//===----------------------------------------------------------------------===//
+#ifndef ONNC_ANALYSIS_GLOBALSTATISTICS_H
+#define ONNC_ANALYSIS_GLOBALSTATISTICS_H
+#include <onnc/Analysis/Statistics.h>
+#include <onnc/Analysis/StatisticsGroup.h>
+
+namespace onnc {
+
+namespace global {
+
+Statistics* GlobalStat();
+
+
+} // namespace of global
+
+
+/// Clear global::config()
+void ClearConfigure();
```

Comment:
How about change the name to `global::ClearStats()`. And let users know this function is just calling `global::stats()->clear()` by inline functions.

However, rememer to add `inline` keyword when you are using inline function in a header.

Reply:
Modify Statistics API according to reviewer's suggestion, and add an example of clear commit: @2f06a3